File: 3138/3138.cs
public class Solution {
    public int MinimumLengthOfAnagramConcaten(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(MinimumLengthOfAnagramConcaten(root.left), MinimumLengthOfAnagramConcaten(root.right));
    }
}

File: 3138/3138.go
func minimumLengthOfAnagramConcaten(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := minimumLengthOfAnagramConcaten(root.Left)
	r := minimumLengthOfAnagramConcaten(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}